docs/linux/helper-function/bpf_redirect_peer.md

```diff
@@ -30,6 +30,10 @@ The helper returns **TC_ACT_REDIRECT** on success or **TC_ACT_SHOT** on error.
 !!! example "Docs could be improved"
     This part of the docs is incomplete, contributions are very welcome
 
+!!! note
+    [:octicons-tag-24: v6.15](https://github.com/torvalds/linux/commits/c4327229948879814229b46aa26a750718888503)
+    With this patch, bpf_redirect_peer now calls skb_scrub_packet. pkt_type is set to PACKET_HOST by default.
```

Comment:
```suggestion
    With this patch, `bpf_redirect_peer` now calls `skb_scrub_packet`. `pkt_type` is set to `PACKET_HOST` by default.
```

Reply:
We should put these in code blocks since they refer to code, and so the spellcheck ignores these.